modules/apps/feature-flag/feature-flag-test/build.gradle

```diff
@@ -0,0 +1,11 @@
+dependencies {
+	testCompileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "default"
+
+	testIntegrationImplementation group: "com.liferay.portal", name: "com.liferay.portal.impl", version: "default"
+	testIntegrationImplementation group: "javax.portlet", name: "portlet-api", version: "3.0.1"
+	testIntegrationImplementation group: "org.apache.felix", name: "org.apache.felix.http.servlet-api", version: "1.1.2"
+	testIntegrationImplementation group: "org.osgi", name: "org.osgi.service.cm", version: "1.6.0"
+	testIntegrationImplementation project(":apps:xstream:xstream-configurator-api")
+	testIntegrationImplementation project(":core:osgi-service-tracker-collections")
+	testIntegrationImplementation project(":test:arquillian-extension-junit-bridge")
```

Comment:
Can you check to see whether all of these dependencies are necessary here? I'm guessing probably get rid of some of them.

Reply:
Ops, turns out I just needed 2 of them :sweat_smile: